duckduckgo/BrowserServicesKit
# Merge PR workflows into a single workflow

**Required**:

Task/Issue URL:  https://app.asana.com/0/1203301625297703/1205270007589105/f
iOS PR:  N/A
macOS PR:  N/A
What kind of version bump will this require?: N/A

**Description**:
* Merge unit-tests.yml and swiftlint.yml to a single pr.yml so it is consistent with other repos
* Add post merge integration with asana 

**Steps to test this PR**:
1. Create a new PR from this branch
2. Remove develop branch check and add code errors
3. Check if task is created under BSK section on https://app.asana.com/0/1205237866452338/1205239782816625 project

Example task created: https://app.asana.com/0/1205237866452338/1205304872646644/f
